Kitodo/src/main/java/org/kitodo/config/Parameters.java

```diff
@@ -0,0 +1,688 @@
+     * ==== ----------------------------------- Directories
+     * -----------------------------------
+     */
+
+    // DIR_XML_CONFIG = "KonfigurationVerzeichnis" → org.kitodo.config.Config
+
+    /**
+     * Absolute path to the directory that the rule set definition files will be
+     * read from. It must be terminated by a directory separator ("/").
+     */
+    public static final String DIR_RULESETS = "RegelsaetzeVerzeichnis";
+
+    /**
+     * Absolute path to the directory that XSLT files are stored in which are
+     * used to transform the "XML log" (as visible from the XML button in the
+     * processes list) to a downloadable PDF docket which can be enclosed with
+     * the physical binding units to digitise. The path must be terminated by a
+     * directory separator ("/").
+     */
+    public static final String DIR_XSLT = "xsltFolder";
+
+    // DIR_PROCESSES = "MetadatenVerzeichnis" → org.kitodo.config.Config
+
+    /**
+     * Absolute path to the base directory of the users' home directories,
+     * terminated by a directory separator ("/"). If a user accepts a task to
+     * work on which will require him or her to have access permission to the
+     * data of a process, a symbolic link to the process directory in question
+     * will be created in his or her home directory that will be removed again
+     * after finishing the task. Note: If LDAP is used, the users' home dirs
+     * will instead be read from LDAP
+     */
+    public static final String DIR_USERS = "dir_Users";
+
+    /**
+     * Absolute path to a folder the application can temporarily create files
+     * in, terminated by a directory separator ("/").
+     */
+    public static final String DIR_TEMP = "tempfolder";
+
+    /**
+     * Path to directory in which BPMN diagrams are stored.
+     */
+    public static final String DIR_DIAGRAMS = "diagramsFolder";
+
+    // DIR_MODULES = "moduleFolder" → org.kitodo.config.Config
+
+    /**
+     * Points to a folder on the file system that contains <b>legacy</b>
+     * Production plug-in jars. In the folder, there must be subfolders named as
+     * defined in enum PluginType (currently: “import” and “opac”) in which the
+     * plug-in jars must be stored.
+     *
+     * <p>
+     * Must be terminated by the file separator.
+     */
+    public static final String DIR_PLUGINS = "pluginFolder";
+
+    /**
+     * Points to a folder on the file system that <b>legacy</b> plug-ins may use
+     * to write temporary files.
+     */
+    public static final String DIR_PLUGINS_TEMP = "debugFolder";
+
+    /*
+     * ----------------------------------- Directory management
+     * -----------------------------------
+     */
+
+    /**
+     * Boolean, defaults to {@code false}.
+     */
+    public static final String CREATE_ORIG_FOLDER_IF_NOT_EXISTS = "createOrigFolderIfNotExists";
+
+    // "createSourceFolder" → org.kitodo.filemanagement.FileManagement
```

Comment:
Why did you add this out commented line? If this line is not needed, then do not add them or add at least an explanation to it.